packages/esm-patient-banner-app/src/banner/patient-banner.component.tsx

```diff
@@ -53,6 +53,11 @@ const PatientBanner: React.FC<PatientBannerProps> = ({
       !(overFlowMenuRef?.current && overFlowMenuRef?.current.contains(event.target)) && onClick(patientUuid);
     }
   };
+  const [showDropdown, setShowDropdown] = React.useState(false);
+  const diappear = React.useCallback((event: MouseEvent) => {
```

Comment:
```suggestion
  const closeDropdownMenu = React.useCallback((event: MouseEvent) => {
```
I believe this is a typo, naming it as `closeDropdownMenu` is more descriptive of the intent of what the function does